stripe/src/test/java/com/stripe/android/model/CardTest.java

```diff
@@ -507,5 +507,52 @@ public void last4ShouldBeNullWhenNumberIsNull() {
         Card card = new Card(null, null, null, null);
         assertEquals(null, card.getLast4());
     }
+
+    @Test
+    public void getLast4_whenNumberIsNullButLast4IsSet_returnsCorrectValue() {
+        Card card = new Card(
+                null,
+                2,
+                2020,
+                "123",
+                "John Q Public",
```

Comment:
Adding tests to make sure we respect the values that we didn't used to respect (when you pass in a brand or a last4, those used to get obliterated if there was no number)